src/index.ts

```diff
@@ -0,0 +1,19 @@
+export class Challenge 
+{
+    buzz(num: number): Promise<string | number> 
+    {
+        return new Promise<string | number>((resolve, reject) => 
+        {
+            let res: string | number = num;
```

Comment:
`res` is not a valid name for a variable.
You should consider using somethig as 'result'.
Another idea is not to use the variable at all.